bleak/backends/corebluetooth/client.py

```diff
@@ -111,7 +111,7 @@ async def connect(self, **kwargs) -> bool:
                 self._peripheral
             )
 
-        def disconnect_callback():
+        def disconnect_callback() -> bool:
```

Comment:
Shouldn't this be `None` instead of `bool`?

Reply:
Yes, good catch! Fixed.